feedhenry-templates/helloworld-android-gradle
# Adding in code coverage

# Adding in Code coverage reports
## Steps to test

``` bash
./gradlew clean connectedCheck 
firefox app/build/reports/coverage/debug/index.html 
```

Observe that you see a code coverage report reading 50 ish percent coverage.
## Notes

<strike>JUnitJacocoTestRunner is a class I found to work around a known bug in the test runner.  Google claims it will be fixed in test runner 0.4.  Test runner 0.4 is published, but it requires build tools 23 (and this Android M).  When Android M support is made available in Digger we can upgrade the template and remove the workaround.</strike>

Found a workaround to use the fixed libs and stay on Android 22.
